yarn.lock

```diff
@@ -1,11662 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
No need to add this file as well.

Reply:
i have put it in git ignore after this pr